token-metadata/js/src/state.ts

```diff
@@ -1,9 +1,15 @@
-import { getArrayDecoder, getBytesDecoder, getStructDecoder, getTupleDecoder } from '@solana/codecs-data-structures';
-import { getStringDecoder } from '@solana/codecs-strings';
-import { TlvState } from '@solana/spl-type-length-value';
 import { PublicKey } from '@solana/web3.js';
-
-import { TokenMetadataError } from './errors.js';
+import {
+    getArrayDecoder,
+    getArrayEncoder,
+    getBytesDecoder,
+    getBytesEncoder,
+    getStructDecoder,
+    getStructEncoder,
+    getTupleDecoder,
+    getTupleEncoder,
+} from '@solana/codecs-data-structures';
+import { getStringDecoder, getStringEncoder } from '@solana/codecs-strings';
 
 export const TOKEN_METADATA_DISCRIMINATOR = Buffer.from([112, 132, 90, 90, 11, 88, 157, 87]);
```

Comment:
I think it still makes sense to expose this even though it is not used, since i understand its a standardised value?

Reply:
Yeah sure.